tests/run-make/link-arg/rmake.rs

```diff
@@ -0,0 +1,13 @@
+use run_make_support::{rustc, tmp_dir};
+
+fn main() {
+    let out = rustc()
```

Comment:
With the [new run-make API](https://github.com/rust-lang/rust/pull/126121), this would looks like this:
```rust
let out = rustc()
        .link_arg("-lfoo")
        .link_arg("-lbar")
        .print("link-args")
        .input("empty.rs")
        .run();
out.assert_stdout_contains("lfoo");
out.assert_stdout_contains("lbar");
```
+ `assert_stdout_contains` should be added [here](https://github.com/rust-lang/rust/blob/0a190e8d2d9bbe0b417b6f160042b816b31cec7b/src/tools/run-make-support/src/command.rs#L114-L113).